src/components/formComponentTemplateTxnOps.tsx

```diff
@@ -0,0 +1,216 @@
+import { JSX } from "react";
```

Comment:
I separated endpoints and operation components because the same param required a different type of component. The logic is more clear now, even though there will be some duplication in code.